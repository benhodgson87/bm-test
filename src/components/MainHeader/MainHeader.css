.main-header {
  margin-bottom: 4px;
  background-color: #fff;
}

.main-header-add {
  display: flex;
  justify-content: flex-end;
  padding: 10px 20px;
}
