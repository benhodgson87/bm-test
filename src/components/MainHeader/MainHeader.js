import React from 'react'

const MainHeader = () => (
  <header>
    <button>Add</button>
  </header>
)

export default MainHeader
