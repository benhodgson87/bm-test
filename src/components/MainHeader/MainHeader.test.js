import React from 'react'
import { shallow } from 'enzyme'

import MainHeader from './MainHeader'

describe('MainHeader component', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<MainHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
