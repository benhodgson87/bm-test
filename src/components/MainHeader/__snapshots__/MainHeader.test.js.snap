// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader component renders correctly 1`] = `
<Row
  className="main-header"
>
  <Col
    className="main-header-add"
    xs={12}
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="primary"
      disabled={false}
    >
      Add New
    </Button>
  </Col>
</Row>
`;
